DownloadAllSongsAndAlbums.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+mkdir -p $HOME/Music/SongLibrary
+mkdir -p $HOME/Pictures/Covers
+cd $HOME/Music/SongLibrary
+
+export AWS_DEFAULT_REGION=us-west-2
+export AWS_DEFAULT_OUTPUT=text
+
+readarray -t song <<< "$(aws s3 ls s3://ucsolarteam.onboardmedia.songs | awk '{for (i=4; i<NF; i++) printf $i " "; print $NF}' | sed s/\ /\+/g | rev | cut -c 5- | rev )"
+albumlist=()
+
+for i in "${song[@]}"; do
+	wget https://s3-us-west-2.amazonaws.com/ucsolarteam.onboardmedia.songs/$i.mp3
+done 
+
+album="$(eyeD3 ${song[0]}.mp3 | grep -o -P '(?<=: ).*(?=year)')"
```

Comment:
It looks like this block is also inside the while loop later on, how come it's out here as well?